tufaceous-lib/src/artifact.rs

```diff
@@ -283,5 +328,18 @@ fn read_entry<R: io::Read>(
 }
```

Comment:
I suspect `read_entry()` should confirm that these are `tar::EntryType::Regular` (i.e., file) entries before trying to extract them.

Reply:
Yeah, totally missed this before, thanks.